perception/radar_fusion_to_detected_object/README.md

```diff
@@ -0,0 +1,75 @@
+# radar_fusion_to_detected_object
+
+This package is radar-based sensor fusion module to 3d detected object.
```

Comment:
What about something like this:
```
This package contains a  sensor fusion module for radar-detected objects and 3D Lidar-detected objects. The fusion node can:

- Attach velocity to 3D Lidar-  detections when successfully matching radar data. The tracking modules use the velocity information to enhance the tracking results while planning modules use it to execute actions like adaptive cruise control.
- Improve the low confidence  Lidar 3D detections when corresponding radar detections are found.
```

Reply:
fix

Reply:
Can you please write the commit where you fixed it?